javaparser-testing/src/test/java/com/github/javaparser/ast/expr/NameTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2007-2010 Júlio Vilmar Gesser.
- * Copyright (C) 2011, 2013-2016 The JavaParser Team.
+ * Copyright (C) 2011, 2013-2017 The JavaParser Team.
```

Comment:
That's a task I would like to participate in. I need that tool too :). I think also about a tutorial for this case, so we can 'kill two birds with one stone' as we are saying in germany :P